tests/console/wicked_basic.pm

```diff
@@ -23,6 +23,12 @@ sub assert_wicked_state {
     assert_script_run("for dev in /sys/class/net/!(lo); do grep \"$status\" \$dev/operstate || (echo \"device \$dev is not $status\" && exit 1) ; done");
 }
 
+sub save_and_upload_log {
+    my $log_name = join('', map { ("a".."z")[rand 26] } 1..8);
```

Comment:
Don't we want to add some prefix and maybe .log extension, so in assets it's clear what kind of file it is?

Reply:
prefix currently added automagically by upload_logs function ( it is module name ). About ".log" - I just added it

Reply:
Yeah, so it will be wicked_basic_%random%.log So if we upload anything else, we have to look into it. I suggest that we put "journal" in name.